prep-javascript-conditionals/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Conditionals</title>
+</head>
+
+<body>
+  <script>
+    // Your code here
+    var age = 22;
+    if (age < 18) {
+      console.log('a Child');
+    } else {
+      console.log('an Adult');
+    }
+    var student1Score = 50;
+    if (student1Score >= 70) {
+      console.log('Congrats! You Received a Passing Grade!');
+    } else {
+      console.log('Oh no you Failed!');
+    }
+    var student2Score = 95;
+    if (student2Score < 60) {
+      console.log('F');
+    }
+    if (student2Score < 70) {
+      console.log('D');
+    }
+    if (student2Score < 80) {
+      console.log('C');
+    }
+    if (student2Score < 90) {
+      console.log('B');
+    }
+    if (student2Score < 100) {
+      console.log('A');
+    } else {
+      console.log('A++');
+    }
+
+    var season = 'Autumn';
+
+    if (season === 'Summer') {
+      console.log("It's hot today");
+    } else if (season === 'Spring') {
+      console.log('The Flowers are blooming');
+    } else if (season === 'Autumn') {
+      console.log('the leaves are changing colors');
+    } else if (season === 'Winter') {
+      console.log("It's cold today");
+    } else {
+      console.log('Please Enter a Valid Season');
+    }
+
+    var dayOfTheWeek = 'Thursday';
+    if (dayOfTheWeek === 'Saturday' | 'Sunday') {
```

Comment:
Here for the `or` operator you need to use 2 `|` pipe characters: `||`. You also need to do the full comparison on both sides, as written this will not work as expected. Here is an example of what it should look like:

```JAVASCRIPT
if (dayOfTheWeek === 'Saturday' || dayOfTheWeek === 'Sunday') {

} else {

}
```